django/forms/models.py

```diff
@@ -319,8 +325,7 @@ def __new__(mcs, name, bases, attrs):
                 message = "Unknown field(s) (%s) specified for %s"
                 message %= (", ".join(missing_fields), opts.model.__name__)
                 raise FieldError(message)
-            # Override default model fields with any custom declared ones
-            # (plus, include all the other declared fields).
+            # Include all the other declared fields.
             fields.update(new_class.declared_fields)
```

Comment:
Do you think there is any way to call this `update` with the remaining of `new_class.declared_fields` and not pass the whole thing which was already (almost) consumed?

Reply:
For this, we'd have to create a new dictionary. Probably not worth doing :thinking: .